CHANGELOG.md

```diff
@@ -53,6 +53,9 @@ Below is an outline of all that is in this release, so you get a sense of all th
 - Now lists codecs from [go-multicodec](https://github.com/multiformats/go-multicodec) library.
 - `ipfs cid codecs --supported` can be passed to only show codecs supported in various go-ipfs commands.
 
+#### `ipfs cid format` command
+- CidV0 Now correctly requires the `dag-pb` codec, instead of the `protobuf` codec.
```

Comment:
Yeah I talked with him about this and he didn't have any concerns, but good to have him double check here.

Reply:
* Re 1:  I think it is fine to support everything from the table: that will allow for experimentation without the need for recompiling go-ipfs or creating custom CID creation tools.  Regular users will only know about IPLD tag ones returned by  `ipfs cid codecs` (+`libp2p-key` for IPNS) so UX is ok.
* Re 2:  yeah.. let's rename `--codec` to `--mc`  (shorter, more aligned with other one-letter params) just to avoid any headache 

As for the changelog, description buries the lead, I propose making it clear what changed:

```suggestion
#### `ipfs cid format` command
- `--codec` was removed and replaced with `--mc` to ensure existing users are aware of below changes:
  - `--mc protobuf`  now correctly points at code `0x50` (was `0x70`, which is `dag-pb`) 
  - `--mc cbor`  now correctly points at code`0x51` (was `0x71`, which is `dag-cbor`) 
```